website/docs/reference/resource-configs/grants.md

```diff
@@ -98,152 +70,45 @@ The `meta` config can also be defined under the `seeds` config block in `dbt_pro
-
+    config:  
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `snapshots` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a snapshot's SQL block
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
-
-<TabItem value="analyses">
-
-The `meta` config is not currently supported for analyses.
-
-</TabItem>
-
-<TabItem value="macros">
-
-<File name='macros/schema.yml'>
-
-```yml
-version: 2
-
-macros:
-  - name: macro_name
-    meta: {<dictionary>}
-
-    arguments:
-      - name: argument_name
-        meta: {<dictionary>}
-
-```
-
-</File>
-
-</TabItem>
-
-<TabItem value="exposures">
-
-<File name='models/exposures.yml'>
-
-```yml
-version: 2
-
-exposures:
-  - name: exposure_name
-    meta: {<dictionary>}
-
-```
-
-</File>
-
-</TabItem>
-
 </Tabs>
 
 ## Definition
-The `meta` field can be used to set metadata for a resource. This metadata is compiled into the `manifest.json` file generated by dbt, and is viewable in the auto-generated documentation.
-
-Depending on the resource you're configuring, `meta` may be available within the `config` property, or as a top-level key. (For backwards compatibility, `meta` is always supported as a top-level key, though without the capabilities of config inheritance.)
-
-<Changelog>
 
-* `v0.16.0`: This property was introduced
-* `v0.21.0`: Introduced the `config` property, and gave `meta` the capabilities of a config
-
-</Changelog>
+You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt, which you can view in the automatically generated documentation.
 
 ## Examples
-### Designate a model owner
-Additionally, indicate the maturity of a model using a `model_maturity:` key.
-
-<File name='models/schema.yml'>
 
-```yml
-version: 2
-
-models:
-  - name: users
-    meta:
-      owner: "@alice"
-      model_maturity: in dev
+Granting a single user a permission:
 
-```
-
-</File>
-
-
-### Designate a source column as containing PII
-
-<File name='models/schema.yml'>
-
-```yml
-version: 2
-
-sources:
-  - name: salesforce
-
-    tables:
-      - name: account
-        meta:
-          contains_pii: true
-        columns:
-          - name: email
-            meta:
-              contains_pii: true
-
-```
-
-</File>
-
-### Configure one meta attribute for all seeds
-
-<File name='dbt_project.yml'>
+```sql
+{{ config(materialized = 'incremental', grants = {
+    'select': 'other_user'
+}) }}
 
-```yml
-seeds:
-  +meta:
-    favorite_color: red
 ```
 
-</File>
-
-### Override one meta attribute for a single model
-
-<File name='models/my_model.yml'>
+Granting multiple users the same permission:
 
 ```sql
-{{ config(meta = {
-    'single_key': 'override'
+{{ config(materialized = 'incremental', grants = {
+    'select': ['other_user','this_user']
```

Comment:
I find the name `'this_user'` to be a bit confusing, because it sounds like the current user that dbt is running as. We should be clear that `grants` are only for granting access to OTHER users/groups/roles

Reply:
I changed to "admin_user" but I'd like a real-world example here.  What would the people do? @dbeatty10 any ideas for me? :)

Reply:
I've seen the following as a real-worldy example for grants:
```'select': ['bi', 'reporter']```